client/src/components/SiteSearch/SiteSearch.global.css

```diff
@@ -0,0 +1,3 @@
+.site-search-section {
+  margin-top: 0;
+}
```

Comment:
I believe we are defaulting to css modules. @wesleyboar that right?

Reply:
We are.

@shayanaijaz was there a reason for this to be global? And, is it still necessary to be global given what you discovered while managing conditional sidebar padding and background color?

Reply:
The styling I want to apply here only needs to be applied to the particular section on the SiteSearch page. Does it make more sense to use css module and have the selector something like this: 

```
:global(.c-section--has-content-layout-has-sidebar.site-search-section ) {
  margin-top: 0;
}
```

Reply:
Nah, that's a bit verbose and has no benefits. **If** it must be global, **then** `...global.css`.

**But**, it does not need to be global. That class is introduced in the code of this PR as global.

<details><summary>Currently, via Global <sub>click to toggle code</sub></summary>

`client/src/components/SiteSearch/SiteSearch.jsx`
```
import './SiteSearch.global.css';

...

    <Section
      className="site-search-section"
      content={
```
`client/src/components/SiteSearch/SiteSearch.global.css`
```
.site-search-section {
  margin-top: 0;
}
```

</details>

**Instead**, that class can be added via CSS module i.e. `...module.css`:

<details><summary>Instead, via Module <sub>click to toggle code</sub></summary>

`client/src/components/SiteSearch/SiteSearch.jsx`
```
import styles from './SiteSearch.module.css';

...

    <Section
      className="styles['root']"
      content={
```
`client/src/components/SiteSearch/SiteSearch.module.css`
```
.root {
  margin-top: 0;
}
```

</details>

I know you've seen a lot of files add styles a lot of different ways. Migration to CSS modules is slow but steady.

Reply:
Ah that makes sense. Thanks for the help!